profile/config.go

```diff
@@ -211,7 +211,7 @@ Examples:
 	if err != nil {
 		return err
 	}
-	cfgOptionBlockInbound = config.Concurrent.GetAsInt(CfgOptionBlockInboundKey, 6)
+	cfgOptionBlockInbound = config.Concurrent.GetAsInt(CfgOptionBlockInboundKey, 4)
```

Comment:
May add a comment why `4` or use a constant

Reply:
Good Point, will update all of them!